subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -466,6 +466,13 @@ config BT_CTLR_SCHED_ADVANCED
 	  Disabling this feature will lead to overlapping role in timespace
 	  leading to skipped events amongst active roles.
 
+config BT_CTLR_ASSERT_OVERHEAD_START
+	bool "Assert on Prepare Latency"
+	default y
+	help
+	  Assert on increased Radio Event Prepare callback latencies due to
+	  CPU usage overheads in the Controller implementation.
```

Comment:
It is unclear to me why this is a Kconfig and not only behavior. What is the side effects of disabling this? (Should be described the by the `help`).